tests/insert_coins.js

```diff
@@ -0,0 +1,97 @@
+const fs = require('fs').promises;
+
+const keyring = new Keyring({
+    type: 'sr25519'
+});
+const nodeAddress = "ws://127.0.0.1:9801";
+
+async function createPromiseApi(nodeAddress) {
+    const wsProvider = new WsProvider(nodeAddress);
+    const api = await ApiPromise.create({
+        provider: wsProvider
+    });
+    await api.isReady;
+    return api;
+}
+
+async function main() {
+    const api = await createPromiseApi(nodeAddress);
+    const sender = keyring.addFromMnemonic("bottom drive obey lake curtain smoke basket hold race lonely fit walk//Alice");
+
+    const mints_file = '/home/georgi/Desktop/workspace/Manta/precomputed-15k-iterations/precomputed_mints_v3-5';
+    const transfers_file = '/home/georgi/Desktop/workspace/Manta/precomputed-15k-iterations/precomputed_transfers_v3-5';
+    const reclaims_file = '/home/georgi/Desktop/workspace/Manta/precomputed-15k-iterations/precomputed_reclaims_v3-5';
+
+    const mints_buffer = await fs.readFile(mints_file);
+    const transfers_buffer = await fs.readFile(transfers_file);
+    const reclaims_buffer = await fs.readFile(reclaims_file);
+
+    const mints_offset = 2;
+    const transfers_offset = 4;
+    const reclaims_offset = 4;
+    const total_iterations = 15000;
```

Comment:
I suppose` total_iterations` can be calculated by transfers_buffer/reclaims_offset/mint_size.
But it's ok to use a hard code here.

Reply:
Will address in #1016